cb-talent-development/employer-style-base
# EM-1021: Style Base Updates - Colors, Buttons, Typography

**Purpose:**
> As a user with trouble seeing color contrast, I need the CareerBuilder site CTAs to be WCAG 2.0 color compliant so I can clearly see what everyone else sees.

Refer to HiringDot_Patterns_5_9_17.pdf on the JIRA ticket

* Color updates
  * Removes brand simplification colors
  * Cleans up shades of grey

| From        | To          |
| ------------- |:-------------:| 
| same      | $grey-darkest: #333 |
| $grey-darker: lighten($grey-darkest, 20); /// #666      | $grey-demon: #666; |
| same | $grey-dark: #999999 |
| same      | $grey-semi-dark: #BBBBBB;|
| $grey-semi-light: #D4D4D4; // border color and $grey-light: #DDDDDD;      | $brand-new-grey-another-grey --> $grey-lighter: #d8d8d8|
| $grey-lighter: #EEEEEE; | $grey-lightest: #F4F4F4; |

  * Adds semantic variables to replace color names (e.g. `$text-highlight`)
  * Adds new gradient mixins
      * wanted to create a function to output gradient mixins but cannot (we could do classes, I prefer to have mixins to add to semantic classes, although the DY stories will probably need classes anyway) - https://github.com/sass/sass/issues/2146
  * Removes `$white` and `$black` as style base variables and instead uses `white` and `black` CSS color keywords

Buttons
* Removes brand simplification styles
* Adds variable for fix-width homepage buttons (`$fixed-button-width`)
* adds all 5 button options (primary through quinary)
* updates button hover states (sans animation)

Type
* Makes body copy 15px
* Susan and I will further map out the header style changes before making them

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/EM-1021

**Changes:**
* Changes to setup, Side effects, Architectural changes, Migrations

* Library changes
  * bumps style base

**Screenshots**
n/a


**QA Links:**
http://web.employer-1.development.c66.me

**How to Verify These Changes**
* Specific pages to visit
  * Home `/`
  * Jobs and Search `/recruiting-solutions/job-postings` and `/recruiting-solutions/resume-database`
  * APS `portal/ppuofkansas`
  * Promotions `/promotions`

* Steps to take
  * Compare branch pages to prod and look for glaring differences; everything should look mostly unchanged

* Responsive considerations
  * n/a


**Relevant PRs/Dependencies:**
https://github.com/cbdr/employer/pull/823

**Additional Information**
